### ISSUE TITLE
[NextBuild] Migrate the [insert chosen feature] to NextBuild

### ISSUE THREAD

**Issue by FranECross** [department-of-veterans-affairs/va.gov-cms#19386]
## Description
The first product feature to be implemented using NextBuild is the Resources & Support content type of Checklist.

The AP team doesn't have capacity at this time to provide focused time or information, so this ticket is for one PW engineer to choose a PW feature to migrate to NextBuild.

The work encompassed in this ticket will entail exploration and laying a lot of groundwork.  Given that templates can share common pieces (such as paragraph types), we might end up with duplicate work, but maybe engineers could pick ones that don't have any overlap, if possible.

Note the addition of a QuickStart guide link below in the Artifacts section.

### User Story or Problem Statement
As a Sitewide engineer, I want my work to use the latest technology and fastest build system, for maximum efficiency and Veteran value. 

As a VA Product Owner, I want my technical products to adopt Accelerated Publishing, to move toward deprecation of content-build as soon as possible. 

As an Editor, I want my content changes to be published as quickly as possible, for maximum Veteran access to the latest correct information. 

## Status / Change management stakeholders
Each Sitewide product will require migration to new templates. Editors who create content in these types will require change management to be aware when the changes are deploying and of the change in behavior for publishing timelines. 

Stakeholder / Editor groups are listed below for visibliity, but each group may require its own sub-epic and change management. 

### Artifacts from Accelerated Publishing Team (in Github)
- [**QUICKSTART GUIDE**](https://github.com/department-of-veterans-affairs/next-build/blob/main/READMEs/quickstart.md0)
- NextBuild [top level page](https://github.com/department-of-veterans-affairs/next-build/tree/42c435d93300b2a541ffb91a1d3aa299e563ef7c#)
- [Example tests](https://github.com/department-of-veterans-affairs/next-build/tree/42c435d93300b2a541ffb91a1d3aa299e563ef7c/example_tests)
- [READMEs](https://github.com/department-of-veterans-affairs/next-build/tree/42c435d93300b2a541ffb91a1d3aa299e563ef7c/READMEs)
- [Meeting recording](https://oddball-io.zoom.us/rec/share/zibAkLSZOahW0GmlUkRyghcO1ULz9NylbF3ZWIlSm4EOZmCvx72AXsa7UpnR5uIU.rXvX4Dbb8DBa_mbZ)

## Acceptance criteria
- [ ] Explore and lay groundwork for migration of [insert feature chosen] to NextBuild
- [ ] Create additional tickets as needed
- [ ] Accessibility review if applicable

**Comment by randimays** [comment]
I have a meeting with Tim this afternoon to work through the remaining Typescript issues I'm having with the project. Those errors may be the reason the web components aren't working correctly (as they cause problems with the build). I believe the TS issues are the last big hurdle to getting this wrapped up.

**Comment by randimays** [comment]
End-of-sprint update: The dev work for this is complete as far as I can tell, and I'm working on getting CI to pass in order for it to be ready to code review. Not sure when the code can be merged / this ticket can close, but it won't be my primary focus going forward as long as there aren't any issues to fix while in code review / functional testing.

**Comment by randimays** [comment]
Another update: I have two PRs in code review for Tim to peek at, so I am done with dev work for now. [The first one](https://github.com/department-of-veterans-affairs/next-build/pull/801) is admin-y and I pulled out some non-functionality changes from my checklist branch for him to review first.

**Comment by randimays** [comment]
End-of-sprint update: Still waiting for code review

**Comment by randimays** [comment]
@jilladams @FranECross 

End of sprint update: I only have one checklist PR remaining. I think the hydration issues with the web components still exist, but I haven't had time to get back to the code and run it again after getting other chunks merged in. Given that the Zenhub -> Github ticket is urgent per the EOY deadline, I'm going to set this aside for now and concentrate on that. This will carry over or we can put it in the backlog if we don't anticipate being able to prioritize this again soon.

**Comment by FranECross** [comment]
@jilladams What do you think of punting this back to the backlog, and picking it back up next year when R&S filtering/tagging is implemented?

**Comment by randimays** [comment]
In an effort to reduce the cognitive load of a giant PR on Tim, I pulled out a lot of little code chunks where things needed to be updated to what is in production (I only did this for things that were used on the checklist template). Those went into a `Sitewide-integration` branch. I'm working on getting that branch merged to `main` now that Events has been successfully released to production.

**However**: I saw [in this Slack thread](https://dsva.slack.com/archives/CT4GZBM8F/p1734037706550309) that removing events from content-build caused some 404s so I think Tim will be busy with troubleshooting that for a little bit. 

Once the `Sitewide-integration` branch has been merged in, I can update my stale checklist branch and get back to figuring out what the hydration issues are caused by and hopefully move the rest of the code into `main` soon.

**Comment by jilladams** [comment]
Components used in R&S content types beyond checklist are in a separate integration branch PR that @timcosgrove  is working through to break into smaller PRs that can merge into main. We think Tim must be the person to do this. This ticket is blocked on the merge of those components, in order for the rest of the work for checklists to merge.

**Comment by jilladams** [comment]
Moved to Admin and marked blocked until we get traction on the blockers from AP.

**Comment by laflannery** [comment]
Closing in favor of the standard epic process the other templates are using for migration
